ui/client/components/graph/NodeDetailsContent.js

```diff
@@ -14,6 +14,8 @@ import Fields from "../Fields";
 import ParameterList from "./ParameterList";
 import ExpressionWithFixedValues from "./ExpressionWithFixedValues";
 
+const VOID_TYPE = "java.lang.Void";
```

Comment:
Instead of this, please modify logic of preparing `returnType` field in schema. Now it is very technical information - it leaks that we have Void/Unit/... markers for situation when no new variable is added to context. I suggest to change this logic to returning null in this case.